packages/react/src/wallet/adapters/WalletGateway.ts

```diff
@@ -41,7 +41,7 @@ export class WalletGateway
   }
 
   async reset(): Promise<void> {
-    await this.storage.set([]);
+    await this.storage.reset();
```

Comment:
if we want to cleanup properly on logout, then better to reset it than leave with empty array